src/extraction/fathom/ruleset_factory.js

```diff
@@ -48,18 +26,18 @@ export default class RulesetFactory {
     // (though we should have distinct penalties for that sort of thing if we
     // care). More importantly, clamp the upper bound of the score so we don't
     // overcome other bonuses and penalties.
-    return linearScale(area, 80 ** 2, 1000 ** 2) ** this.bigImageCoeff;
+    return linearScale(area, 80 ** 2, 1000 ** 2);
   }
 
   /** Return whether the computed font size of an element is big. */
   fontIsBig(fnode) {
     const size = parseInt(getComputedStyle(fnode.element).fontSize, 10);
-    return linearScale(size, 14, 50) ** this.bigFontCoeff;
+    return linearScale(size, 14, 50);
   }
 
   /** Scores fnode with a '$' in its innerText */
   hasDollarSign(fnode) {
-    return (fnode.element.innerText.includes('$') ? ONEISH : ZEROISH) ** this.hasDollarSignCoeff;
+    return (fnode.element.innerText.includes('$') ? ONEISH : ZEROISH);
```

Comment:
We should get rid of ONEISH and ZEROISH. They don't make anything better anymore, and they make make things ever so slightly worse (probably just slightly slower to converge).